server/testserver.go

```diff
@@ -15,24 +15,24 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
+// A TestServer encapsulates an in-memory instantiation of a cockroach
+// node with a single store. Example usage of a TestServer follows:
+//
+//   s := server.StartTestServer(t)
+//   defer s.Stop()
+//
+
 package server
 
 import (
 	"testing"
 
-	"github.com/cockroachdb/cockroach/gossip"
-	"github.com/cockroachdb/cockroach/proto"
-	"github.com/cockroachdb/cockroach/security"
-	"github.com/cockroachdb/cockroach/storage"
-	"github.com/cockroachdb/cockroach/storage/engine"
-	"github.com/cockroachdb/cockroach/util"
-	"github.com/cockroachdb/cockroach/util/hlc"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 // StartTestServer starts a in-memory test server.
-func StartTestServer(t testing.TB) *TestServer {
-	s := &TestServer{}
+func StartTestServer(t testing.TB) *EmbeddedServer {
```

Comment:
The original reasoning for passing a `testing.TB` here was to make sure that nobody would ever confuse this for non-test code (since it's in the package and exported). Since that's not possible after this change why not just panic on an error, and update the comment to stress once more that this is not code intended for anything in production.